include/dlaf/factorization/cholesky/mc/cholesky_L.h

```diff
@@ -11,36 +11,103 @@
 
 #include <hpx/include/parallel_executors.hpp>
 #include <hpx/include/threads.hpp>
-#include <hpx/include/util.hpp>
-#include <hpx/local/future.hpp>
+#include <hpx/util/annotated_function.hpp>
```

Comment:
I would recommend you keep `hpx/include/util.hpp` even though it may include more than necessary. `hpx/util/annotated_function.hpp` is deprecated (or will be in the next release). We haven't decided yet into what smaller public header that functionality would go.